roles/kubespray-defaults/defaults/main/main.yml

```diff
@@ -763,3 +763,15 @@ system_upgrade_reboot: on-upgrade  # never, always
 
 # Enables or disables the scheduler plugins.
 scheduler_plugins_enabled: false
+
+# Symlinks to etcd certs
+kube_etcd_cacert_file: "kube-client-ca.pem"
+kube_etcd_cert_file: "kube-client-cert.pem"
```

Comment:
I think this is going to cause problems with the way we generate certs and key.
They are generated on the first etcd, and I think the reason there is the inventory_hostname in the filename in the first place is to distinguishes them.

See https://github.com/kubernetes-sigs/kubespray/blob/b551fe083d181fc753cc6457d7711ef5f136f036/roles/etcd/tasks/check_certs.yml#L32-L62

Reply:
it doesn't change the certificate generation of etcd. 
They are still created on first node and then propagated on others. 

Those are links to the node-{{inventory_hostname}*

Reply:
Oh right, those variables are not used for the generation itself, I was confused ^.